.gitlab-ci.yml

```diff
@@ -0,0 +1,87 @@
+
+# /************************************************************************
+ #  File: .gitlab-ci.yml
+ #  Author: mdr0id
+ #  Date: 9/10/2018
+ #  Description:  Used to setup runners/jobs for librustzcash
+ #  Usage: Commit source and the pipeline will trigger the according jobs.
+ #
+ #  Known bugs/missing features:
+ #
+ # ************************************************************************/
+  
+stages:
+  - build
+  - test
+  - deploy
+
+rust-latest:
+  stage: build
+  image: rust:latest
+  script:
+    - cargo --verbose --version
+    - rustc --verbose --version
+    - time cargo build --verbose
+
+#Ignore the nightly debug CI job for now (9/18/2018)
+.rust-debug-latest:
+  stage: build
+  image: rust:latest
+  script:
+    - cargo build --debug --verbose
```

Comment:
There is no `--debug` flag. `cargo build` uses debug mode by default, and you turn on release mode with `--release` (which we don't need to a CI system).

Reply:
I was not aware of this convention.